blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: Getting Started with Blazor Predefined Dialogs Component | Syncfusion
```

Comment:
Title: Getting Started with the Blazor Predefined Dialogs Component | Syncfusion